features-json/document-query-command-supported.json

```diff
@@ -0,0 +1,227 @@
+{
+  "title":"Document.queryCommandSupported()",
+  "description":"Reports whether or not the specified editor query command is supported by the browser.",
+  "spec":"https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html#querycommandsupported()",
+  "status":"ed",
```

Comment:
@dandv You'll want `unoff` there. Just updated the contributions file to clarify.